.devops/deploy-pipelines.yml

```diff
@@ -70,3 +70,16 @@ stages:
                     SUBSCRIPTION: PROD-IO-SERVICE-CONN
                     ENVIRONMENT: prod
                     WORKINGDIR: 'src/core'
+      - deployment: migrate
+        continueOnError: false
+        dependsOn: apply
+        environment: 'PROD'
+        strategy:
+          runOnce:
+            deploy:
+              steps:
+                - checkout: self
+                - script: |
+                    cd src/psql/selfcare/subscription-migrations
+                    bash ./flyway.sh migrate PROD-IO
```

Comment:
```suggestion
                    bash ./flyway.sh migrate PROD-IO db
```

Reply:
I actually hardcoded `db` into the shell script - I can move it out again